notebooks/Emukit-multi-fidelity-bayesian-optimization.ipynb

```diff
@@ -0,0 +1,564 @@
Nm3asGjRIu64445U9+NWsGBBGjZsCECNGjWSxFaqVCmaNm3K1KlT6dy5c8L0adOm0bFjRypWrHjF7YuIiOREhw7BBx+8w7Fj+3nxxc8wxvg6pExTC72kS2BgYJJkPiM2bdrE3r176d27N7GxsQmvokWL0qRJE1asWHHFbZQoUSJJUh0YGEj16tXZvXt3quv8/PPPAPTo0SPJ9NTuLlzNPjIiMjKSWrVqJSTzAH5+fini+/7777HW0qtXryTnq2XLlgQEBKTrfKXH8OHDWbhwYcLxrVy5ks2bNzNkyBCvbF9ERORaEx0Nv/xyhi++eJGmTdvSuHFLX4fkFWqh94Jly5alOi8wMDDN+aVLl05zfuXKldOcnx2t8wDlypW76nUPHz4MQK9evejVq1eK+bVr177iNkqWLJliWkBAQJp3CdylPMk7ypYtW9Zr+8iIAwcOeNx38mnu81WpUiWP2zl27JhX4unZsyf/+te/mDZtGuPHj2fKlCmEhIRc9YWbiIjItS4qCr788i1OnjzKsGHjfR2O1yihl3TxdDvK398fPz8/Ll26lGR68oTTPYzka6+9RsuWKa+EE4+b7k3ly5cHnATZ/TPAoUOHsmR/6YlnzZo1KaYnj8d9vpYtW0bhwoVTLF+mTBmvxFOwYEH69+/P9OnTeeihh/jiiy8YM2YM+fLpxp2IiOQ+R4/Cnj1QoUIZOnV6gPr1b/Z1SF6jhF6ump+fHxUrVmTjxo1JpifvaHnDDTdQoUIFtmzZwr/+9a9si89dM/75558n1NC7P/tC8+bN+eyzz1i3bh2NGjUCIC4uLkU87dq1wxjD3r17Pd7RyAj30KIXLlzwOH/o0KG89tpr3HPPPcTHxzNgwIBM7U9ERORaFBMDGzZA8eLQtetAunYd6OuQvEoJvWTKvffey+uvv85LL71EaGgoS5cuTZGg+vn58dZbb9GjRw/Onz/P3XffTalSpTh48CA//fQTtWvX5qGHHvJ6bA0aNODuu+/mqaeeIjo6mkaNGvH999+zaNEigGxviR44cCCvvPIKnTp1YsKECZQoUYLJkycTHR2dZLnrrruORx99lMGDB7Nx40Zuu+02AgIC2L17N4sWLeLhhx/mlltuSdc+K1WqRNGiRfnkk0+oU6cOgYGB1KhRgxIlSgBQvXp1wsPD+e677+jRo0eq5UgiIiI52d9/w9mzF1m3bg7t2t1D/vz+vg7Jq3RvXTJl7NixDBo0iDfeeIOuXbuyY8cOZsyYkWK5rl27snTpUo4fP86AAQMIDw9n1KhRHD16lJtvzrpbXh988AG9evXihRdeoFu3bmzfvp1JkyYBUKxYsSzbryeFChXihx9+4LrrrmPw4MH079+f66+/PslINm6vv/46b775Jj/88APdu3enS5cuvPrqq5QpU4Zq1aqle5/+/v689957HDx4kH/84x80adKE77//Psky7k656gwrIiK50alTsH07/PTTDMaM6cO6dT/6OiSvM7lhMH1vCQ0NtZ5qnN22bNmS8KAeybnGjx/P2LFj2b9/v1qkgbvvvpuNGzcmPBAsq+nvSEREsktcHPz8M0RHx9C7dy1Kly7P9OmrfD5U5bFjEB5ecIO1Fxt4Y3squZFcbd68eWzbto0GDZy/l+XLl/P666/Tp0+fPJ3MW2v55ZdfWL16NfPmzUt4yq6IiEhusmcPnD4Nv/76KQcO7OLJJ9/Klf/fKaGXXK1IkSJ8/vnnjB8/nvPnz1O5cmUef/xxxowZ4+vQfCo6OprmzZtTpEgRhgwZwsCBuatzkIiIyLlzsHUrFCsWx/TpE6hduyEtWnTwdVhZQgm95Gpt2rShTZs2vg7jmlOwYEFUbiciIrmVtbB5MxQoACdPHqBAgQD69x+dK1vnQQm9iIiIiOQyBw86484HB0OxYpX49NN1vg4pS2mUGxERERHJNaKjYeNGKFEC9u/fydmzp8iXL1+ufnBi7j2yLKIyBZGrp78fERHJalFRzru/P7z00oP06RNKfHy8b4PKYkroM8Df3z/VJ26KyJVduHABf//c9TAPERG5dhw7Bnv3Oq3zf/+9iZ9++pYOHfrm6tZ5UEKfIcHBwezbt4/z58+rpVEkA6y1nD9/nn379hEcHOzrcEREJBeKjYX166FYMTAGPv74VQoWDKR792G+Di3LqVNsBhQtWhSA/fv3ExMT4+NoRHIWf39/ypYtm/B3JCIi4k07dsClS1C0KBw+vI/vvvuEbt2GULx4KV+HluWU0GdQ0aJFlZCIiIiIXENOn4Zt26B0aefzjz8uID4+jvvuG+nbwLKJEnoRERERybHi42HTJggMBHepfLdug2nWrB0VKoT4NLbsohp6EREREcmx9u6FkychKMj5HBvrlEXnlWQelNCLiIiISA514QJs2QIlSzqf4+LiuOeeG5g+fYJvA8tmSuhFREREJMex1knm8+d3XgArV37Drl1RVKlSy7fBZTMl9CIiIiKS4xw+DAcOQPHil6fNnv1fypatTFhYV98F5gNK6EVEREQkR7l0CTZudB4g5bZt2wZWr15Cjx4Pkj9/3hr3RQm9iIiIiOQo27ZBXBwEBFyeNnv2mwQEFKJLl4G+C8xH8tbli4iIiIjkaCdPws6dkPzB4w888G+aNm2XJx4klZwSehERERHJEeLiYMMGKFIEjEk6r0KFkDw1VGViKrkRERERkRxh1y44e9Z5iJRbbGwsY8f2ZePGX3wXmI/liITeGFPJGPOmMSbSGHPeGGONMSEelitojHnFGHPAGHPBtXzL7I9YRERERLzp7FmIioJSySpqli2bx4IFH3H06EHfBHYNyBEJPVAT6AmcAFamsdz7wCBgDNAROAB8b4xplOURioiIiEiWsBY2bYKCBcHPL+m8WbP+S8WK1bjtto6+Ce4akFMS+hXW2rLW2juBzz0tYIxpCNwHjLTWvmut/R/ORcBu4PnsC1VEREREvGn/fjh+HIoWTTp969a1rFv3Iz16PIRf8kw/D8kRCb21
```

Comment:
Let's also cite this one here
https://papers.nips.cc/paper/5086-multi-task-bayesian-optimization.pdf